From 05a6e65a2d7d062f283786660e8e76b514e1b010 Mon Sep 17 00:00:00 2001
From: me-cafebabe <me.cafebabe@gmail.com>
Date: Tue, 31 Jan 2023 17:06:07 +0800
Subject: [PATCH 7/8] Revert "PropsUtils: Use Build.FINGERPRINT instead of
 Build.DATE"

This reverts commit 7eeef908e20faea830443ff69913be99567a65ba.
---
 core/java/com/android/internal/util/miku/PropsUtils.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/com/android/internal/util/miku/PropsUtils.java b/core/java/com/android/internal/util/miku/PropsUtils.java
index a8d91a8cd54f..11bb4845c6dd 100644
--- a/core/java/com/android/internal/util/miku/PropsUtils.java
+++ b/core/java/com/android/internal/util/miku/PropsUtils.java
@@ -102,7 +102,7 @@ public class PropsUtils {
 	}
         // Set proper indexing fingerprint
         if (packageName.equals("com.google.android.settings.intelligence")){
-            setPropValue("FINGERPRINT", Build.FINGERPRINT);
+            setPropValue("FINGERPRINT", Build.DATE);
         }
     }
 
-- 
2.30.2

